extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -88,7 +133,7 @@ Implicit tiling enables procedurally-generated tilesets. Instead of serving stat
 
 ## Tileset JSON
 
-As in [3D Tiles 1.0](https://github.com/CesiumGS/3d-tiles/tree/master/specification#tileset-json), one tileset JSON file is the entry point for defining an implicit tileset. To use implicit tiling, the `3DTILES_implicit_tiling` extension must be defined in one of the tiles of the tileset JSON file. This tile is treated as the root of an implicit tileset, even if it is not the root of the tileset JSON. The implicit root tile must not define any explicit `children`.
+The `3DTILES_implicit_tiling` extension may be defined on any tile in the tileset JSON file. Such a tile is called an **implicit root tile**, to distinguish it from the root node of the tileset JSON. The implicit root tile must not define the `children` property.
```

Comment:
Rereading the document again I think it could benefit from having a high level technical overview before jumping into the JSON example, something that describes everything in maybe 4 sentences like https://github.com/CesiumGS/3d-tiles/blob/metadata-revisions/extensions/3DTILES_metadata/1.0.0/README.md#overview